src/common/legacy/src/transformations/convert_opset1_to_legacy/convert_tile_to_ie_tile.cpp

```diff
@@ -81,8 +81,13 @@ ngraph::pass::ConvertTileToLegacyMatcher::ConvertTileToLegacyMatcher() {
             ++tiles_it;
         }
 
-        last_node.get_node_shared_ptr()->set_friendly_name(tile->get_friendly_name());
-        ngraph::copy_runtime_info(tile, new_ops);
+        // In case new_ops is empty it means that IE Tile was not created and tile is replaced with input.
+        // In name of the input will be change there will be issue with conversion from ngraph::Function to CNNNetwork.
+        if (!new_ops.empty()) {
+            last_node.get_node_shared_ptr()->set_friendly_name(tile->get_friendly_name());
```

Comment:
I think we should update name of last operation anyway and update rt info for last_node

Reply:
In current master to network which have following layers Parameter->Broadcast it is transformed to
Parameter->Tile.
At the end Tile should be transformed to TileIE otherwise will be rejected by function converting ngraph::Function to CNNNetworw.

last_host is initialized with input of Tile which is Parameter layer(which is also input of the network).

In case there will be no Tile which meets following condition `if (tile_dim != 1) {` 0 there is no new layer created and last_node is pointing to Parameter.  
In current master in such case Parameter will be renamed with with name of Tile ( initial name of Broadcast) 
and will have copied runtime_info of Tile. Finally Tile is replaced with last_node.


The convertFunctionToICNNNetwork() function is looking for inputs in CNNNetwork and ngraph::Function which should be Parameter but it cannot match it which leads to segmentation fault.
I have fixed it to throw the exception instead a segfault, but still I thought it would be good avoid situation that input names for CNNNetwork and ngraph::Function does not match. 

@nkogteva 
Do you have any guidance how to fix conversion function in better way?

Reply:
@marcinkusm Just to clarify: is this Tile doing nothing (since it is just removed)?

Reply:
It is how I understand current code in `master` branch in case following condition is never met `if (tile_dim != 1) {`